### ISSUE TITLE
putout formatter completely breaks static classes

### ISSUE THREAD

**Issue by ElPrudi** [coderaiser/putout#159]
I really, really dislike that the putout formatter runs automatically when you want to fix a simple rule with this plugin. One of the worst offenders is, that it turns this:

```ts
abstract class X {
    static #x?: boolean = true
    private static z: number = 5

    public static a(): boolean {
       this.#x = true
       const y = true
       return this.#x ? y : false
   }
}
```

to this:

```ts
abstract class X {
    #x: boolean = true 
    z: number = 5

    a() {
       this.#x = true
       const y = true
       return this.#x && y
   }
}
```

Just because it found this:
```
Simplify ternary (simplify-ternary/value)                               putout/putout
```

So it:
- removes all `static` keywords
- removes all `public`, `private` and `protected` keywords
- removes all return type definitions
- removes all `?` that indicate optional properties

So, please give us an option to turn off the putout formatter. It's way too destructive.

**Comment by coderaiser** [comment]
Thank you! Just fixed in [`@putout/printer@2.85`](https://github.com/putoutjs/printer/releases/tag/v2.85.0) 🎉 . Please re-install 🐊.

Is it works for you?

You feedback really helps, anyways you can always [switch](https://github.com/coderaiser/putout#printer) to `recast` with:

```js
{
    "printer": "recast"
}
```

It works slower than `@putout/printer` and has pure maintenance, but it more mature for now , anyways `@putout/printer` is much easier to support, and all regular JavaScript/JSX/TypeScript cases are supported (or will be when I have code examples 😏).

**Comment by ElPrudi** [comment]
Yes, setting it to `recast` fixed everything. Thank you very much :)
